htdocs/product/inventory/inventory.php

```diff
@@ -1045,8 +1045,9 @@ function barcodeserialforproduct(tabproduct,index,element,barcodeproductqty,sele
 		// Expected quantity = Quantity in stock when we start inventory
 		print '<td class="right expectedqty" id="id_'.$obj->rowid.'" title="Stock viewed at last update: '.$obj->qty_stock.'">';
 		$valuetoshow = $obj->qty_stock;
+
 		// For inventory not yet close, we overwrite with the real value in stock now
-		if ($object->status == $object::STATUS_DRAFT || $object->status == $object::STATUS_VALIDATED) {
+		if (($object->status == $object::STATUS_DRAFT || $object->status == $object::STATUS_VALIDATED) && empty($conf->global->DISABLE_QTY_OVERWRITE)) {
```

Comment:
Can you replace with
!getDolGlobalString('DISABLE_QTY_OVERWRITE');

Reply:
Yes, it's done :)